recipe/build.sh

```diff
@@ -1,43 +1,91 @@
 #!/bin/bash
 
+set -x
 
-for USE_WIDEC in false true;
-do
-    WIDEC_OPT=""
-    if [ "${USE_WIDEC}" = true ];
-    then
-        WIDEC_OPT="--enable-widec"
-    fi
+# ncurses (gen-pkgconfig.in) adds -ltinfo to ncurses.pc if any of the following conditions is true:
+# 1. No `*-Wl,-rpath,*` is found in EXTRA_LDFLAGS
+# 2. Any `*--as-needed*` is found in EXTRA_LDFLAGS
+# The build system takes care that any `-Wl,-rpath,` in LDFLAGS gets copied into EXTRA_LDFLAGS
+# (and also that any -L${PREFIX}/lib gets translated to -Wl,-rpath,${PREFIX}/lib)
+# the same is not true of -Wl,--as-needed (which is referenced only in gen-pkgconfig.in).
+# One option to fix this is to pass our LDFLAGS as EXTRA_LDFLAGS however this ends up copying across
+# all of our linker flags into the .pc file which means they are forced upon all pkg-config based
+# consumers of ncurses and that is a very bad thing indeed. If we wanted to do that we would:
+# export EXTRA_LDFLAGS=${LDFLAGS}
+# export LDFLAGS=
+# .. but instead it is better to strip off all '-Wl,-rpath,*' and '-L${PREFIX}' from LDFLAGS.
+re='^(.*)(-Wl,-rpath,[^ ]*)(.*)$'
+if [[ ${LDFLAGS} =~ $re ]]; then
+  export LDFLAGS="${BASH_REMATCH[1]}${BASH_REMATCH[3]}"
+fi
+re='^(.*)(-L[^ ]*)(.*)$'
+if [[ ${LDFLAGS} =~ $re ]]; then
+  export LDFLAGS="${BASH_REMATCH[1]}${BASH_REMATCH[3]}"
+fi
+
+./configure \
+  --prefix="${PREFIX}" \
+  --build=${BUILD} \
+  --host=${HOST} \
+  --without-debug \
+  --without-ada \
+  --without-manpages \
+  --with-shared \
+  --disable-overwrite \
+  --enable-symlinks \
+  --enable-termcap \
+  --with-pkg-config-libdir="${PREFIX}"/lib/pkgconfig \
+  --enable-pc-files \
+  --with-termlib \
+  --enable-widec
```

Comment:
Am concerned that not providing the non-widec (normal) libraries may break downstream users. Is there a reason not to provide them?